src/Microsoft.ML.Data/Model/Repository.cs

```diff
@@ -10,11 +10,6 @@
 
 namespace Microsoft.ML.Model
 {
-    /// <summary>
-    /// Signature for a repository based model loader. This is the dual of ICanSaveModel.
-    /// </summary>
-    public delegate void SignatureLoadModel(ModelLoadContext ctx);
-
```

Comment:
I guess the reason of removing this is that we choose `ICanSaveModel` as the universal way to dump a trained model.

Reply:
No I just moved it to the `ModelLoading` file.

---
In reply to: [254961987](https://github.com/dotnet/machinelearning/pull/2431#discussion_r254961987) [](ancestors = 254961987)